scanners/ssh-audit/parser/parser.js

```diff
@@ -0,0 +1,262 @@
+            description: "Discouraged SSH message authentication code algorithms in use",
+            hint: "Remove these MAC algorithms"
+        },
+        enc: {
+            name: "Insecure SSH Encryption Algorithms",
+            description: "Discouraged SSH Encryption algorithms are in use",
+            hint: "Remove these encryption algorithms"
+        }
+    },
+    delWarning: {
+        kex: {
+            name: "Insecure SSH KEX Algorithms",
+            description: "Discouraged SSH key exchange algorithms in use",
+            hint: "Remove these KEX algorithms"
+        },
+        key: {
+            name: "Insecure SSH Key Algorithms",
+            description: "Discouraged SSH key algorithms in use",
+            hint: "Remove these key algorithms"
+        },
+        mac: {
+            name: "Insecure SSH MAC Algorithms",
+            description: "Discouraged SSH message authentication code algorithms in use",
+            hint: "Remove these MAC algorithms"
+        },
+        enc: {
+            name: "Insecure SSH Encryption Algorithms",
+            description: "Discouraged SSH Encryption algorithms are in use",
+            hint: "Remove these encryption algorithms"
+        }
+    },
+    chgCritical: {
+        kex: {
+            name: "SSH KEX Algorithms must be changed",
+            description: "Weak SSH key exchange algorithms in use",
+            hint: "Change these KEX algorithms"
+        },
+        key: {
+            name: "SSH Key Algorithms must be changed",
+            description: "Weak SSH key algorithms in use",
+            hint: "Change these key algorithms"
+        },
+        mac: {
+            name: "SSH MAC Algorithms must be changed",
+            description: "Weak SSH message authentication code algorithms in use",
+            hint: "Change these MAC algorithms"
+        },
+        enc: {
+            name: "SSH Encryption Algorithms must be changed",
+            description: "Weak SSH encryption algorithms in use",
+            hint: "Change these encryption algorithms"
+        }
+    },
+    chgWarning: {
+        kex: {
+            name: "SSH KEX Algorithms must be changed",
+            description: "Weak SSH key exchange algorithms in use",
+            hint: "Change these KEX algorithms"
+        },
+        key: {
+            name: "SSH Key Algorithms must be changed",
+            description: "Weak SSH key algorithms in use",
+            hint: "Change these key algorithms"
+        },
+        mac: {
+            name: "SSH MAC Algorithms must be changed",
+            description: "Weak SSH message authentication code algorithms in use",
+            hint: "Change these MAC algorithms"
+        },
+        enc: {
+            name: "SSH Encryption Algorithms must be changed",
+            description: "Weak SSH encryption algorithms in use",
+            hint: "Change these encryption algorithms"
+        },
+
+        addCritical: {
```

Comment:
This looks weird, the `addCritical` seems to be inside the `chgWarning` object. 
Is that really intended or a typo?

Reply:
Yes it was a typo, thank you!